lib/components/icons/Icon/Icon.tsx

```diff
@@ -2,8 +2,8 @@ import React, { ComponentType } from 'react';
 import classnames from 'classnames';
 import { TextSize, Fill, IconSize } from '../../../themes/theme';
 import { Box } from '../../Box/Box';
-import { ThemeConsumer } from '../../ThemeConsumer/ThemeConsumer';
```

Comment:
Can we delete the `ThemeConsumer` entirely now?

Reply:
Good question